libs/common/src/enums/feature-flag.enum.ts

```diff
@@ -17,6 +17,7 @@ export enum FeatureFlag {
   EnableDeleteProvider = "AC-1218-delete-provider",
   ExtensionRefresh = "extension-refresh",
   RestrictProviderAccess = "restrict-provider-access",
+  BulkDeviceApproval = "AC-1710-bulk-device-approval",
```

Comment:
I think you probably got this key from my server PR, but I actually had to change it. The key EMs made is just `bulk-device-approval`.
```suggestion
  BulkDeviceApproval = "bulk-device-approval",
```